src/i_video.c

```diff
@@ -483,12 +483,23 @@ static void UpdateMouseButtonState(unsigned int button, boolean on, unsigned int
     D_PostEvent(&event);
 }
 
+static event_t hold_event;
```

Comment:
Could you also add a small comment here, please? I am pretty sure this will be non-obvious to me in two years. 😉